pieforproviders/pieforproviders
# Adds a 404 page, updates Rails fallback index

## 📋 Summary
Adds a 404 page, updates Rails fallback index, and cleans up the React Router a bit to allow for 404s
Ticket: Fixes #22 404s on Deployed envs

## 🧐 Why tho
Smarter handling of routing so users get expected behavior when hitting a found route or a not-found route.  Hopefully this will handle rails' propensity to 404 on reload of Dashboard but I'm not 100% on that.

## 💻 Steps to set up locally
- restart both servers

## ✅ Steps to test
- visit `localhost:3000` and get redirected to `/login`
- visit `localhost:3000/dashboard` and see the dashboard get rendered
- visit `localhost:3000/randomString` and see a 404
